modules/dataframe/src/jsondataframeconversion.cpp

```diff
@@ -0,0 +1,154 @@
+            // Not supported
+            break;
+        case json::value_t::array:  ///< array (ordered collection of values)
+            // Not supported
+            break;
+        case json::value_t::string:  ///< string value
+            df.addCategoricalColumn(header, 0u);
+            break;
+        case json::value_t::boolean:  ///< boolean value
+            // FIXME: Get compile error when trying to add bool column
+            // Error C2039 'data' : is not a member of 'std::vector<bool,std::allocator<_Ty>>' 
+            // \inviwo\core\datastructures\buffer\bufferramprecision.h 155	
+            //df.addColumn<bool>(header, 0u);
+            df.addColumn<int32_t>(header, 0u);
+            break;
+        case json::value_t::number_integer:  ///< number value (signed integer)
+            df.addColumn<int32_t>(header, 0u);
+            break;
+        case json::value_t::number_unsigned:  ///< number value (unsigned integer)
+            df.addColumn<uint32_t>(header, 0u);
+            break;
+        case json::value_t::number_float:  ///< number value (floating-point)
+            df.addColumn<float>(header, 0u);
+        case json::value_t::discarded:  ///< discarded by the the parser callback function
+            break;
+    }
+}
+
+}  // namespace detail
+
+void to_json(json& j, const DataFrame& df) {
+    for (auto row = 0; row < df.getNumberOfRows(); ++row) {
+        json node = json::object();
+        auto items = df.getDataItem(row, true);
+        // Row 0 in the dataframe contains the row indices, which is not needed in the json object.
+        int i = 1;
+        for (auto col = ++items.begin(); col != items.end(); ++col) {
+            node[df.getHeader(i++)] = (*col)->toString();
+        }
+        j.emplace_back(node);
+    }
+}
+
+void from_json(const json& j, DataFrame& df) {
+    // Extract header and column types
+    if (j.empty() || !j.front().is_object()) {
+        // Only support object types, i.e. [ {key: value} ]
+        return;
+    }
+    // Extract header names from first object
+    const auto& firstRow = j.at(0);
+    for (const auto& col : firstRow.items()) {
+        switch (col.value().type()) {
+            case json::value_t::null: {  ///< null value
+                // need to check more rows to know type
+                auto c = std::distance(firstRow.items().begin(), col);
+
+                for (const auto& row : j) {
+                    auto item = row.find(col.key());
+                    if (item->type() != json::value_t::null) {
+                        detail::addDataFrameColumnHelper(item.value().type(), col.key(), df);
+                        // Stop searching when we found a non-null item
+                        break;
+                    }
+                }
+                break;
+            }
+            case json::value_t::object:  ///< object (unordered set of name/value pairs)
+                // Not supported
```

Comment:
Should the print some warning / throw error, just silent ignore seems potentially bad / confusing